### DISCUSSION TITLE
Automill goes slightly under the piece

### DISCUSSION THREAD

**Discussion by pagliaso** [fellesverkstedet/Bark-beetle-parametric-toolpaths#89] [General]
Hi Jens,

I hope you could have some rest!

When I do 3D shape with Automill and generate Cutout, the path goes slightly under the piece into the bed, like 0,6 mm or so.

I wonder if I am doing something wrong, or if this is the expected behaviour.

I did some more tests with G21 and it is going great.

Thanks and kind regards,

Alb

**Comment by JensDyvik** [comment]
Hi. I am sorry I missed this question. Good to hear that G21 worked. The automill going 0,6mm deeper on cutouts is actually my design choice, but it is possible to adjust the value to 0 quite easily. Would you like me to look up where the setting is and post a screenshot?

**Reply by aeltobv** [reply]
Hi Jens, I happen to have the same question. It would be great if you could explain where we can edit the value of 0.6mm to 0 for example? 

Thanks!

**Comment by JensDyvik** [comment]
No problem

First you need to find the automill cluster:
![auto1](https://github.com/user-attachments/assets/b2a8e2d6-55c7-4d82-9fa1-b2225b6fad1d)

Double click the cluster to enter:
![auto2](https://github.com/user-attachments/assets/c6a10076-e0a0-4904-8852-a980526e8da0)

At the beginning you find the panel with the extra depth setting:
![auto3](https://github.com/user-attachments/assets/4aaab835-4c5f-4a13-8a17-39efbf2caeb0)

Change the value from 0.6 to 0 (don't press enter, click outside the panel to save the new value):
![auto4](https://github.com/user-attachments/assets/ec1a098c-049b-4d1c-83ca-c6f8216cea26)

Press this button and choose "save and close":
![auto5](https://github.com/user-attachments/assets/ac789e67-7d77-4613-9f23-21d73be97116)

For those of you who are new to BarkBeetle, you can enable cutouts from automill by default in the very bottom of all the settings:
![auto6](https://github.com/user-attachments/assets/8229afa4-891b-4522-95be-8a41e9520ce0)

Thank you for your appreciation of BarkBeetle. It definitely helps motivate me for starting work on BarkBeetle 2.0 :)
